doc/src/user_guide.rst

```diff
@@ -537,6 +537,31 @@ Example::
     min-fact = 0.3
     max-fact = 2.5
 
+[solver-dual-time-integrator-multip]
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Parameterises multi-p for dual time-stepping with
+
+1. ``pseudo-dt-fact`` --- factor by which the pseudo time-step size
+   changes between multi-p levels:
+
+    *float*
+
+2. ``cycle`` --- nature of a single multi-p cycle:
+
+    ``[(`` *int* ``,`` *int* ``), (`` *int* ``,`` *int* ``), ... (``
```

Comment:
Level is meaningless, it is a term we have never used before.  Order, as in polynomial order, is unambiguous.